src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcAccountMenu.scala.html

```diff
@@ -69,50 +69,12 @@
     </li>
 
     <li>
-      <a href="#subnav-your-account" id="account-menu__main-2" class="hmrc-account-menu__link hmrc-account-icon hmrc-account-menu__link--more" aria-haspopup="true" aria-owns="subnav-your-account">
+      <a href="@yourProfile.href" class="hmrc-account-menu__link@if(yourProfile.active){ hmrc-account-menu__link--active}else{}">
```

Comment:
Do we need the `else` block here?

Reply:
ah maybe not, we just copy pasted from the others I think, we can try it out

Reply:
that worked, we've remove it from this and the other places 👍